index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
And had this been a PR that would be accepted this would actually be a `major` change because it did not `adding functionality in a backwards-compatible manner`